### ISSUE TITLE
In excel 2013 the TaskPanes will be shared bug

### ISSUE THREAD

**Issue by 571821151** [NetOfficeFw/NetOffice#227]
![image](https://user-images.githubusercontent.com/20980198/47546084-2bb1fb00-d922-11e8-975b-3bd689e9f26e.png)
if i tap the button in a new excel ,it will show the TaskPane in old excel document.
![1](https://user-images.githubusercontent.com/20980198/47546379-97e12e80-d923-11e8-8028-1439b9278e70.gif)

i find that in excel 2010,it is normal .

**Comment by jozefizso** [comment]
Yes, even when you try the Excel sample **02 Ribbons And Panes** it will only display the Task Pane in a single Excel window.

If you use the NetOffice `[CustomPaneAttribute]` to define a task pane, NetOffice will create only a single instance of it and it will be bound to the first application window.

To create multiple task panes you must create task panes manually.

https://docs.microsoft.com/en-us/office/vba/api/Office.CustomTaskPane
